src/frontend/get-jira-configuration.ts

```diff
@@ -56,14 +56,14 @@ export const getFailedConnections = (installations, subscriptions) => {
 		.map((failedConnection) => ({
 			...failedConnection,
 			...subscriptions.find(
-				(sub) =>
-					failedConnection.id === sub.gitHubInstallationId
+				(sub) => failedConnection.id === sub.dataValues.gitHubInstallationId
 			),
 		}))
 		.map((sub) => {
-			const repos = sub.reposSyncState || {};
+			const repos = sub?.dataValues?.repoSyncState?.repos || {};
 			const repoId = Object.keys(repos);
-			const orgName = repos[repoId[0]].repository?.owner.login || undefined;
+			const orgName =
+				repos[repoId[0]] && repos[repoId[0]].repository?.owner.login || undefined;
```

Comment:
could use optional chaining.  Also, why are we only getting the first one?  What happens if there's no repos?

Reply:
There's a test for this https://github.com/atlassian/github-for-jira/blob/cec00d3ebf86b2cd02cf69343185f6663710b659/test/frontend/util/get-jira-configuration-helpers.test.ts#L86